src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -214,8 +214,116 @@ As AWS is a paying service, when the Scheduler is stopped normally (without remo
 
+==== AWS Autoscaling Infrastructure
 
+Similarly to the EC2 Infrastructure, the *AWS Autoscaling Infrastructure* operates AWS EC2 service to provide computing node to the Resource Manager.
+However, it implements a different instance management from it to lower the delay of node acquisition and node removal process and ease inter-node collaboration in the same cluster.
 
+	1. The instances operating the nodes are allocated from a common instance template.
+
+	2. The nodes share the same networking infrastructure through a common VPC.
+ The infrastructure support networking autoconfiguration if no parameter is supplied.
+
+===== Pre-Requisites
+
+The configuration of the AWS Autoscaling infrastructure is subjected to several requirements.
+
+	1. The administrator needs both an *AWS access key* and an *AWS secret access key* to enable proactive to authenticate against AWS.
+ Please refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html?icmpid=docs_iam_console#Using_CreateAccessKey[the AWS documentation] to learn dof to get them.
+
+	2. The *AWS region* that will support the node cluster must not have reach its resources quota.
+ It has to be able to allocate one instance template in every case situation.
+ If the network autoconfiguration has to be be triggered, the region has to be able to provide one VPC, one subnet, one internet gateway and one security group.
+ The number of maximum allocated instances is to be configured by the administrator, but has also to comply with the limitation of the regions.
+ Detailed informations are available in AWS documentation for https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#limits_vpc[VPC] and https://aws.amazon.com/fr/ec2/faqs/#How_many_instances_can_I_run_in_Amazon_EC2[Instances]. 
+
+	3. The administrator must be in possession of a valid *AWS keypair* and the ID of the *Amazon Machine Image (AMI)* to provision instances operating ProActive node.
+ If the AMI to use does not propose the Linux operating system, the administrator must supply a *provision script* to (i) download a Java Runtime Environment (JRE), download Proactive Node agent (node.jar), and start up the ProActive agent. 
+
+	4. Optionally, if networking autoconfiguration is not triggered, the administrator has to configure (i) a public *VPC*, (ii) a *Subnet* complying with with VPC CIDR configuration, (iii) an Internet gateway for that VPC and a (iv) Security Group authorizing HTTPS connection to Internet and PNP or PAMR connection to the Resource Manager.
+ The administrator must be in possession of the IDs of the VPC, of the Subnet and of the Security Group.
+
+	5. The Resource Manager has to be accessible from the AWS cloud.
+ Please consider replacing PNP by PAMR as communication protocol if the Resource Manager is located behind a NAT gateway. 
+
```

Comment:
We could add link to the section 6 (`_installation_on_a_cluster_with_firewall`) to explain the PAMR protocol.

Reply:
Done